### PR TITLE
removed chalk as it will no longer work with this project.

### PR DESCRIPTION
**Pull request by DetroitWhiskey136**
also edited the moddep file to show a proper error when a module fails to load.

**Please describe the changes this PR makes and why it should be merged:**

**Status**

- [ ] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [ ] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: actions/aaa_dbmmods_dependencies_MOD.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ const Mods = {
       require('child_process').execSync(`npm i ${moduleName}`);
       try {
         resolve(require(moduleName));
-      } catch {
-        console.error(`Failed to Install ${moduleName}, please re-try or install manually with "npm i ${moduleName}"`);
+      } catch (error) {
+        console.error(error);
```
Comment (almeidx) [inline actions/aaa_dbmmods_dependencies_MOD.js:10]: Or something similar 
```suggestion
        console.error(`Failed to install ${moduleName}. Error: ${error.message}`);
```

**Review by almeidx — CHANGES_REQUESTED** [review]
Why was the colour support removed from Send Message to Console?

**Comment by DetroitWhiskey136** [comment]
chalk uses ESM now and i really couldn't be bothered to figure out how to use imports in the project

**Comment by DetroitWhiskey136** [comment]
For more info see [Chalk](https://www.npmjs.com/package/chalk)

**Comment by generalwrex** [comment]
You don't need that module if you look at my other stuff I implemented the
coloring without any modules

**Comment by almeidx** [comment]
> chalk uses ESM now and i really couldn't be bothered to figure out how to use imports in the project

Replace `require('chalk')` with `await import('chalk')`

File: actions/aaa_dbmmods_dependencies_MOD.js [hunk 2]
```diff
@@ -183,101 +183,13 @@ const Mods = {
...
-        this.dispatchers[id]._eventSetup = true;
-      }
-    };
-  },
 };
 
 module.exports = {
   name: 'Mods',
-  section: 'JSON Things',
+  section: '#DBM Mods',
```
Comment (almeidx) [inline actions/aaa_dbmmods_dependencies_MOD.js:190]: This should be updated in docs file

File: actions/mod_info_MOD.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
   name: 'Welcome',
-  section: '#Mod Information',
+  section: '#DBM Mods',
```
Comment (almeidx) [inline actions/mod_info_MOD.js:3]: This should be updated in docs file

Comment (almeidx) [reply actions/mod_info_MOD.js:3]: Also seems like there are some links pointing to the old repository URL. Should probably update those

**Review by almeidx — APPROVED** [review]

**Pull request merged**